src/main/java/apptive/devlog/Member/Controller/MemberController.java

```diff
@@ -0,0 +1,40 @@
+package apptive.devlog.Member.Controller;
+
+import apptive.devlog.Global.Auth.Dto.SignUpDto;
+import apptive.devlog.Global.Response.Result.ResultCode;
+import apptive.devlog.Global.Response.Result.ResultResponse;
+import apptive.devlog.Member.Domain.Member;
+import apptive.devlog.Member.Dto.UpdateProfileDto;
+import apptive.devlog.Member.Dto.UpdateProfileRequest;
+import apptive.devlog.Member.Service.MemberService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.nio.file.AccessDeniedException;
+
+@RestController
+@RequestMapping("/user")
```

Comment:
리소스는 복수형으로 표현하는 것이 좋습니다
```suggestion
@RequestMapping("/users")
```